src/main/flight/pid_luxfloat.c

```diff
@@ -54,147 +54,149 @@
         const int32_t stickPosAil = ABS(getRcStickDeflection(ROLL, rxConfig->midrc));
         const int32_t stickPosEle = ABS(getRcStickDeflection(PITCH, rxConfig->midrc));
         const int32_t mostDeflectedPos =  MAX(stickPosAil, stickPosEle);
 
         // Progressively turn off the horizon self level strength as the stick is banged over
         horizonLevelStrength = (float)(500 - mostDeflectedPos) / 500;  // 1 at centre stick, 0 = max stick deflection
-        if(pidProfile->H_sensitivity == 0){
+        if(pidProfile->D8[PIDLEVEL] == 0){
             horizonLevelStrength = 0;
         } else {
-            horizonLevelStrength = constrainf(((horizonLevelStrength - 1) * (100 / pidProfile->H_sensitivity)) + 1, 0, 1);
+            horizonLevelStrength = constrainf(((horizonLevelStrength - 1) * (100 / pidProfile->D8[PIDLEVEL])) + 1, 0, 1);
         }
     }
 
     // ----------PID controller----------
     for (int axis = 0; axis < 3; axis++) {
-        SET_PID_LUX_FLOAT_LOCALS(axis);
-        // -----Get the desired angle rate depending on flight mode
-        uint8_t rate = controlRateConfig->rates[axis];
+        const uint8_t rate = controlRateConfig->rates[axis];
 
+        // -----Get the desired angle rate depending on flight mode
+        float angleRate;
         if (axis == FD_YAW) {
             // YAW is always gyro-controlled (MAG correction is applied to rcCommand) 100dps to 1100dps max yaw rate
-            AngleRate = (float)((rate + 10) * rcCommand[YAW]) / 50.0f;
-         } else {
-            // calculate error and limit the angle to the max inclination
+            angleRate = (float)((rate + 27) * rcCommand[YAW]) / 32.0f;
+        } else {
+            // control is GYRO based for ACRO and HORIZON - direct sticks control is applied to rate PID
+            angleRate = (float)((rate + 27) * rcCommand[axis]) / 16.0f; // 200dps to 1200dps max roll/pitch rate
+            if (FLIGHT_MODE(ANGLE_MODE) || FLIGHT_MODE(HORIZON_MODE)) {
+                // calculate error angle and limit the angle to the max inclination
+                // multiplication of rcCommand corresponds to changing the sticks scaling here
 #ifdef GPS
-             const int32_t errorAngle = (constrain(rcCommand[axis] + GPS_angle[axis], -((int) max_angle_inclination),
-                    +max_angle_inclination) - attitude.raw[axis] + angleTrim->raw[axis]) / 10.0f; // 16 bits is ok here
+                const float errorAngle = constrain(2 * rcCommand[axis] + GPS_angle[axis], -((int)max_angle_inclination), max_angle_inclination)
+                        - attitude.raw[axis] + angleTrim->raw[axis];
 #else
-             const int32_t errorAngle = (constrain(rcCommand[axis], -((int) max_angle_inclination),
-                    +max_angle_inclination) - attitude.raw[axis] + angleTrim->raw[axis]) / 10.0f; // 16 bits is ok here
+                const float errorAngle = constrain(2 * rcCommand[axis], -((int)max_angle_inclination), max_angle_inclination)
```

Comment:
@borisbstyle , the `2*` multiplier is there to max luxfloat the same as mwrewrite, and mwrewrite has the `2*` multiplier. Why the original mwrewrite has the multiplier and the original luxfloat does not, I don't know.